linode/manager
# upcoming: [M3-7902] - Add PG region limit help text

## Description 📝
Simple UI change in the `PlacementGroupsCreateDrawer` to inform the user of the Placement Group limits based on selected region.

## Changes  🔄
List any change relevant to the reviewer.
- Conditional logic to render the Placement Group limits.

## Target release date 🗓️
04/15/2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![a1a1](https://github.com/linode/manager/assets/119514965/730bf53b-3dda-436b-a2e8-5043e425a915) | ![a2a2](https://github.com/linode/manager/assets/119514965/24c658f2-a9ef-46b5-a762-f2aa2b90a8a7) |


## How to test 🧪
### Prerequisites
(How to setup test environment)
- Log in with a user account which has the `placement-group` customer tag.
- Using the Cloud Manager developer tools, switch to the `dev` environment.

### Verification steps
(How to verify changes)
- Verify that the text copy is rendered upon selecting a region from the dropdown in the `PlacementGroupsCreateDrawer` component. 
- Close the drawer and verify the text is not rendered until a region is selected.


## As an Author I have considered 🤔

*Check all that apply*
